noty-android/app/composeapp/src/main/java/dev/shreyaspatil/noty/composeapp/view/screen/LoginScreen.kt

```diff
@@ -54,28 +56,29 @@ import kotlinx.coroutines.ExperimentalCoroutinesApi
 @Composable
 fun LoginScreen(navController: NavHostController, loginViewModel: LoginViewModel) {
 
-    val viewState = loginViewModel.authFlow.collectAsState(initial = null).value
-
-    when (viewState) {
+    when (val viewState = loginViewModel.authFlow.collectAsState(initial = null).value) {
         is ViewState.Loading -> LoaderDialog()
         is ViewState.Failed -> FailureDialog(viewState.message)
         is ViewState.Success -> {
             navController.navigate(
                 route = Screen.Notes.route,
                 builder = {
                     launchSingleTop = true
+                    popUpTo(Screen.Login.route) {
+                        inclusive = true
+                    }
                 }
             )
         }
     }
 
-    LazyColumn {
+    LazyColumn(
+        modifier = Modifier
+            .fillMaxSize()
+            .background(Color.White)
+    ) {
```

Comment:
This change isn't working I guess. Please look into it.

<img src="https://user-images.githubusercontent.com/19620536/107871516-601d9c80-6ec8-11eb-86a0-c5decbcad73b.jpg" height="300"/> <img src="https://user-images.githubusercontent.com/19620536/107871518-61e76000-6ec8-11eb-8a26-09e0bcf05113.jpg" height="300"/>

Reply:
I missed that fix on Signup page, will push the changes for all